projects/plugins/jetpack/changelog/add-remove-regression

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Fix regression from https://github.com/Automattic/jetpack/pull/31200/files
```

Comment:
I think we can make this more user friendly

Reply:
Done